### ISSUE TITLE
mkvtoolnix section

### ISSUE THREAD

**Issue by ablwr** [amiaopensource/ffmprovisr#411]
Something that came up during the most recent No Time To Wait conference was that it'd be useful to add a section of recipes for mkvtoolnix.

I support this idea. I would be interested to hear from others what commands they semi-regularly use or find useful, and we can implement them in a section ala `imagemagick` and `flack`.

CC: @bturkus

**Comment by retokromer** [comment]
That’s certainly useful. We actually work often with MKVToolNix.

**Comment by kieranjol** [comment]
Yeah, specifying colour properties is useful and also inserting chapter
markers. Extracting attachments is also a bit awkward to figure out so
having a handy recipe here could be useful for rawcooked users.

**Comment by bturkus** [comment]
Yeah love all that. I’ve found mkvinfo (or mkvmerge -i; can’t remember!) an impt piece of this; you need to get correct track/attachment numbers before doing anything else. 

My inability to remember these commands proves the need for this. Great idea Ashley!

**Comment by ablwr** [comment]
Thank Stephen McConnachie!

**Comment by kieranjol** [comment]
Yeah,these commands are difficult to remember for sure. I always have to
look them up each time.

**Comment by kfrn** [comment]
Good idea!
